### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by TomaszEichler**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
-int fibonacci_iterative(int sequence) {
+int fibonacci_iterative(int sequence){
     // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0){
+        return 0;
+    }
+    else if(sequence == 1){
+        return 1;
+    }
```
Comment (mariomoore) [inline module1/homework/fibonacci/fibonacci.hpp:10]: It works fine, but you can also write it this way:
```
if (sequence < 2) {
    return sequence;
}
```

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
-    // TODO: Your implementation goes here
-    return 0;
+int fibonacci_recursive(int sequence){
+
+    if(sequence == 0){
+        return 0;
+    }
+    else if(sequence == 1){
+        return 1;
+    }
```
Comment (mariomoore) [inline module1/homework/fibonacci/fibonacci.hpp:31]: What do you think about:
```
if (sequence < 2) {
    return sequence;
}
``` ?

File: module1/homework/fibonacci/fibonacci.hpp (hunk 1 above)
Comment (TomaszEichler) [reply module1/homework/fibonacci/fibonacci.hpp:10]: Yes I know, I did it consciously. The most common description of this algorithm separates these two cases so I did the same. Anyway thank you :)

File: module1/homework/fibonacci/fibonacci.hpp (hunk 2 above)
Comment (TomaszEichler) [reply module1/homework/fibonacci/fibonacci.hpp:31]: Same here :)

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**